app/org/sagebionetworks/bridge/services/SendMailViaAmazonService.java

```diff
@@ -87,7 +87,7 @@ public void sendConsentAgreement(User user, ConsentSignature consentSignature, S
         try {
```

Comment:
So, do we want the birth date sent as part of the participant report, so the consent admin can prove everyone declared themselves of age.